challenge.js

```diff
@@ -9,7 +9,7 @@
  */
 
 // ANSWER
-
+You jump to your development tools and open the console, pass the element id/class into defined variables through DOM, then make changes(add in this case) new elements to the page.
```

Comment:
Sure, that's right - you'll grab an element through `document`, using one of its `getElement(s)` methods, and add a new element as a property to one of the resulting elements.